packages/rlc-common/src/buildIndexFile.ts

```diff
@@ -297,7 +299,9 @@ function generateRLCIndex(file: SourceFile, model: RLCModel) {
     ]);
   }
 
-  if (hasPollingOperations(model)) {
+  // TODO: Disable LRO poller tentatively and remember to remove this when new LRO in RLC is ready
+  // https://github.com/Azure/autorest.typescript/issues/2230
+  if (hasPollingOperations(model) && !model.options?.isModularLibrary) {
```

Comment:
> I wonder if we still need to split the cjsName and esModuleName if with Jose's esm PR merged?
I may not get the point completely, but we still need the cjs pattern for RLC in swagger way.

> I wonder if we should consider cases where we have lro operations in RLC, but no LRO operation in modular due to client.tsp's config?
The case should have no impact for this pr, because modular use its own helper to build LRO and for long term we would split RLC and modular code and they could calculate by themselves.